apps/prairielearn/src/pages/instructorAssessmentInstances/instructorAssessmentInstances.js

```diff
@@ -106,6 +106,7 @@ router.post(
         requireOpen,
         close,
         overrideGradeRate,
+        null, //client_fingerprint_id
```

Comment:
Formatting nit (we conventionally put leading spaces on comments):

```suggestion
        null, // client_fingerprint_id
```

Reply:
Added this leading space in commit 52d0e1a.